installer/conf/omsagent.conf

```diff
@@ -89,6 +89,19 @@
   retry_wait 30s
 </match>
 
+<match oms.operation.**>
+  type out_oms
+  log_level info
```

Comment:
Default log level is usually warning although there is nothing logged in `operation_lib.rb`